benchmarks/quant-hela-method-comparison/scripts/02_prepare_peptides.py

```
@@ -0,0 +1,513 @@
+    if feature_path.exists():
+        try:
+            if feature_path.suffix == ".parquet":
+                print(f"  Loading parquet: {feature_path.name}")
+                return pd.read_parquet(feature_path)
+            elif feature_path.suffix == ".csv":
+                print(f"  Loading CSV: {feature_path.name}")
+                return pd.read_csv(feature_path)
+            elif feature_path.suffix == ".tsv":
+                print(f"  Loading TSV: {feature_path.name}")
+                return pd.read_csv(feature_path, sep="\t")
+        except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)